components/nexus-blobstore-file/src/main/java/org/sonatype/nexus/blobstore/file/MapdbBlobMetadataStore.java

```diff
@@ -437,6 +444,28 @@ public long getTotalSize() {
   }
 
   @Override
+  public long getBlobSize() {
+    ensureStarted();
+    return database.execute(new Fun.Function1<Long, DB>()
+    {
+      @Override    
+      public Long run(final DB db) {
+        long totalSize = 0l;
```

Comment:
Note that `0l` and `0L` are both primitive long literals, it's just that `0L` is more readable.